packages/dev-scripts/bin/sf-nuts.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const shell = require('../utils/shelljs');
+
+const packageRoot = require('../utils/package-path');
+const { resolveConfig } = require('../utils/sf-config');
+
+const nyc = require.resolve('nyc/bin/nyc');
+
+const config = resolveConfig(packageRoot);
+const testConfig = config.test || {};
+const includes = testConfig.testsPath || '**/*.nut.ts';
+
+const command = `${nyc} mocha "${includes}"`;
```

Comment:
Would we want to standardize some of the likely mocha options for nuts?
--timeout 600000 --parallel

Reply:
Having parallel as a default would be good for unit tests, not so sure for nuts.

Reply:
Not a big fan of global timeouts, that should be at the discretion of the test writer. My work thus far on org nut tests has setup creating two new orgs and the test typically takes more than five minutes to complete against my local build. The `it` part of the test is fairly quick, but setup is the drag on runtime.

We may get better perf against production.

Reply:
prod: 7 commands executed across 2 orgs each (not parallel), 2 minutes.

Reply:
@mshanemc what's the timeout override going to look like?

Reply:
Yes!  Thanks I forgot about that.

Reply:
@peternhale our current .mocharc has timeout at 5000.  So we'll have to override that.

Doing a global sets a reasonable default that test owners can modify at the test/describe/it level using normal mocha 

Otherwise, without a default, the test will fail until they do them all by hand.

You can override a bunch of levels if you want longer/shorter https://mochajs.org/#timeouts